scripts/run_cu_md_schnet.py

```diff
@@ -0,0 +1,68 @@
+import os
+import sys
+
+import numpy as np
+
+sys.path.insert(0, os.getcwd())
```

Comment:
Not sure if this script runs as is for you, but I needed to modify this line to 
`sys.path.insert(0, os.path.join(os.path.dirname(__file__), ".."))` for it to work.

Reply:
Is there a reason why we're avoiding installation of `ocpmodels` (e.g. `pip install -e .` from the repository root)? Doing this should prevent the need to hack the Python path

Reply:
@calebho No reason really, that's a good suggestion :) we can do that.

I'm not super familiar with how that handles updates in the codebase. Would we need to do a `pip install -e .` after every code change during development?

Reply:
No—the `-e` flag means editable, meaning that `pip` will install it into the library path (`site-packages`) using a what's effectively a symlink to your checkout of the repo. (As an aside, if you omit `-e`, then you'd need to reinstall after every code change)

Reply:
Got it, thanks! That sounds perfect. Will make the change.